new-release/index.ts

```diff
@@ -18,6 +18,7 @@ class NewReleaseAction extends Action {
 			getRequiredInput('labelColor'),
 			getRequiredInput('labelDescription'),
 			+getRequiredInput('days'),
+			getInput('oldVersionMessage'),
```

Comment:
Maybe I'm confused but when do we use `getRequiredInput` and `getInput` because here it's marked required but we just use `getInput`

Reply:
the YML file isn't actually used by anything. I think maybe the rich editor uses it in theory, but it doesn't actually work AFAIK. Removed required line there anyway